tests/sql/moz-fx-data-shared-prod/telemetry_derived/clients_first_seen_v2/moz-fx-data-shared-prod.telemetry_derived.clients_daily_v6.schema.json

```diff
@@ -112,5 +112,9 @@
   {
     "type": "STRING",
     "name": "normalized_os_version"
+  },
```

Comment:
are you going to be adding the actual colum to clients_daily in this PR too? if so, how we aggregate still probably needs to be discussed. we previously said to take the value from the first main ping, but based on @SuYoungHong 's recent analysis, we might want to consider just checking whether any ping has ` startup_profile_selection_reason_first = 'firstrun-created-default'`. Because the first arriving ping might not be the first _generated_ ping. TBD

Reply:
see discussion in this [gdoc comment](https://docs.google.com/document/d/1mG43ZLMcGcLBnJEPw7mxPIQHGvp_S8T56pnT6lvSXXo/edit?disco=AAAA96heEEM)

Reply:
`clients_daily` was updated in https://github.com/mozilla/bigquery-etl/pull/4473, I'll follow up in the doc